src/PKSim.BatchTool/Services/JsonSimulationRunner.cs

```diff
@@ -37,7 +37,7 @@ public class JsonSimulationRunner : IBatchRunner
          _commandTask = commandTask;
       }
 
-      public Task RunBatch(dynamic parameters)
+      public async Task RunBatch(dynamic parameters)
```

Comment:
I have also unified all the export to xxx file to be async. That way they all return task